Upgrade to bootstrap 3.2

Daniel-KM opened the pull request:
Hi,

I upgraded to Bootstrap 3.2. 

Sincerely,

Daniel Berthereau
Infodoc & Knowledge management

Commit:
Beginning cleanup for the new Bootstrap v.3 branch

gsbodine committed:
Converted font awesome to CDN

gsbodine committed:
Working draft of collapsible main navigation. Added brand to config.

gsbodine committed:
A couple of header updates

Daniel-KM committed:
Merge branch 'master' into bootstrap3

Conflicts:
	README.md

Daniel-KM committed:
Removed images of Martha Bery Digital Archive.

Daniel-KM committed:
Fixed search form and javascript.

Daniel-KM committed:
Added an option to load Bootstrap from the theme.

Daniel-KM committed:
Upgraded items/browse to Omeka 2.

Daniel-KM committed:
Upgraded main views for Bootstrap 3.

Daniel-KM committed:
Added banner for a temporary message.

Daniel-KM committed:
Fixed some views, Exhibits in particular.

Daniel-KM committed:
Disabled Google Analytics by default.

Daniel-KM committed:
Fixed container.

Daniel-KM committed:
Upgraded to Bootstrap 3.2.

gsbodine commented:
Thanks Daniel! I've moved to other things lately and let this drop. It's much appreciated.

Daniel-KM committed:
Adapted for upgrade of Bootstrap.

Daniel-KM committed:
Upgraded to Bootstrap 3.3.1.

Daniel-KM committed:
Fixed exhibits/browse.

regan008 committed:
First swing at the theme:
	-rearranged nav to include search form, links, and logos all in one line
	-adjusted padding to search for and nav links
	-added gradient to search bar
	-added webfonts and styled h1-h6
	-styled links and buttons

regan008 committed:
switched from bootstrap 2.3 to bootstrap 3 classes. Fixes to the nav bar to accomodate logo

Commit:
added font awesome icons. Updated show.php with font awesome icon classes.

Commit:
changes to fonts and item metadata layout.

Commit:
Upgrading to Bootstrap v.3.3.4

Commit:
Updated classes on the prev/next item navigation to reflect bootstrap 3.3.4.  Made minor styling/padding changes.  Began to format the exhibit page in exhibits/show.php

Commit:
Styling exhibit pages and navigation

Commit:
fixes to logo and nav

Commit:
added files for Slider Pro (http://bqworks.com/slider-pro/#example1) and initialized on homepage. Still needs some styling and adjustment -- especially for the image size.

Commit:
Slider fixes. Might want to shift tabs to the right side still. TBD. Added tagline and functions for the recent items, featured exhibit and homepage text.

Commit:
beginning of tag cloud code. Needs to be styled

Commit:
changes to the tag sizes on homepage.

Commit:
added social media to footer, styled tags, various fixes to fontawesome code for tags across site.

Commit:
Added rights, identifier, and contributor to item page. Fixes #3.

Commit:
Added twitter icon to navigation bar. Closes #8.

Commit:
closes #4.

Commit:
styling for exhibit landing page nav. Closes #7.

Commit:
Removed twitter icon from footer and made omeka props smaller. Related to #8.

Commit:
colored the collapse icon and fixed the class so it works in mobile view.  Fixes #10.

regan008 committed:
Switched to new slider and formatted it on the `index.php` and in `footer.php`. Styled featured exhibit and added new row below tag line. Removed about section.

regan008 committed:
fixing some styling on homepage featured exhibit. Added map to the homepage

regan008 committed:
adding images

regan008 committed:
Merge pull request #14 from regan008/mosaic-slider

Mosaic slider

Commit:
Merge branch 'master' of https://github.com/regan008/WearingGayHistoryTheme

Commit:
Adding some additional fonting to make 'Lato' font default. Fixed coloring of nav tabs on browse page and tags color of browse and item pages. Adjusted padding on the item page's bottom next/prev navigation. Fixes #11 and #5.

Commit:
adding map marker

Commit:
removing slider pro files no longer needed

Commit:
adding link to map on homepage

Commit:
fixing link to map on homepage

Commit:
Updating readme file. Fixes to the author info on stylesheet.

regan008 committed:
adding jquery to footer so that any child pages underneath about will populate with child pages. Total hack but it works for now.

Commit:
Changes to the readme.md file to credit original authors of libraries. Changed theme.ini info and added new screenshot.

regan008 committed:
removing jquery hack. Broke too  many things.

Commit:
removed jquery hack. Breaks things.

Commit:
removed tags from individual tags to solve duplication in issue #16. #16 closed.

Commit:
Adding google analytics to header. Closes #17.

Commit:
fixed link to loading.gif and removed slider-pro css file link.

Commit:
merged master fixed conflict.

Commit:
reverted tags back to original code from forked branch. Not ideal because the code is trying to use icons that aren't included in theme but will work for now until I figure out a fix.

Commit:
added conditional statement so that slider js/css files are only loaded on the homepage. Fixes #23.

Commit:
various styling for the about page tabs.

Commit:
updating map img

Commit:
added bootstrap styling to exhibits/browse.php page to fix bug where first item sat up next to navigation.

Commit:
adjusting size of map image on homepage

Commit:
added px onto image size.

Commit:
fixing local link to map image on homepage.

Commit:
logo responsiveness fix.

Commit:
added image caption styling

Commit:
caption fixes since Omeka strips out figure and figcaption tags.

Commit:
fixed width on caption

regan008 committed:
adding backup font to body and removing commented out font declaration. Possibly fixes #28.

regan008 committed:
attempting to correct #28.

Commit:
testing url

Commit:
changing how the google fonts are called.

regan008 committed:
added facebook logo. Closes #29

regan008 committed:
added more tags. added view more button. Fixes #27.

regan008 committed:
adjust tag padding in css. add more tags to homepage. Closes #27.

regan008 committed:
Merge pull request #30 from WearingGayHistory/homepageedits

Homepageedits

regan008 committed:
adjusting down number of tags. Related to #27.

Commit:
Added subject to item page. closes #31.

Daniel-KM committed:
Fixed navigation.

Daniel-KM committed:
Updated readme.

Daniel-KM committed:
Upgraded to Bootstrap 3.3.7.

Daniel-KM committed:
Updated theme.ini.

Daniel-KM committed:
Replaced social network links by config options.

Daniel-KM committed:
Updated header for last release of Bootstrap.

Daniel-KM committed:
Added discovery links.

Daniel-KM committed:
Added an option for the grid rotator.

Daniel-KM committed:
Upgraded and internalized fonts awesome to 4.7.0.

Daniel-KM committed:
Removed specific images.

Daniel-KM committed:
Removed specific files of "Wearing Gay History".

Daniel-KM committed:
Renamed beta.css to corner-banner.css.

Daniel-KM committed:
Reset the option for Google Analytics.

Daniel-KM committed:
Typos.

Daniel-KM committed:
Updated modernizer to 3.3.1 (same custom).

Daniel-KM committed:
Upgraded for new version of Geolocation.

Daniel-KM committed:
Fixed exhibits.

Daniel-KM committed:
Genericized form to search items.

Daniel-KM committed:
Included "Bootstrap3-Zend-Framework" to decorate forms.

Daniel-KM committed:
Added ellipsis on title and description inside lists.

Daniel-KM committed:
Reverted manually added form-control and control-label.

Daniel-KM committed:
Fixed tags in item relations advanced search.

Daniel-KM committed:
Added an option to select the display of items/show.

Daniel-KM committed:
Re-enabled the search form.

Daniel-KM committed:
Set a fallback image in list.

Daniel-KM committed:
Removed exhibits/item.php.

Daniel-KM committed:
Updated grid rotator.

Daniel-KM committed:
Kept the "navigation" class for the main nav.

Daniel-KM committed:
Added icon to Simple Contact form.

Daniel-KM committed:
Moved search form javascript into globals.js.

Daniel-KM committed:
Integrated pages for Guest User.

Daniel-KM committed:
Integrated References.

Daniel-KM committed:
Simplified collections/browse.

Daniel-KM committed:
Updated search form to manage old releases.

Daniel-KM committed:
Moved the advanced search partial for map.

Daniel-KM committed:
Genericized items show.

Daniel-KM committed:
Genericized simple contact form.

Daniel-KM committed:
Genericized main menu.

Daniel-KM committed:
Added an option to display tags on homepage.

Daniel-KM committed:
Fixed display group of Bootstrap for Zend.

Daniel-KM committed:
Improved some details and typos.

Daniel-KM committed:
Fixed featured records on the home page.

Daniel-KM committed:
Added total result for collections/browse.

Daniel-KM committed:
Updated version.

Daniel-KM committed:
Removed Google fonts.

Daniel-KM committed:
Moved menu redecoration in libraries.

Daniel-KM committed:
Added support for one level of sub-menus for the navigation bar.

Daniel-KM committed:
Improved display of collections/browse and items/browse.

Daniel-KM committed:
Genericized items/tags.

Daniel-KM committed:
Added a carousel on the home page.

Daniel-KM committed:
Added glyphicon and standardized secondary menu.

Daniel-KM committed:
Added a glyphicon to items/show.

Daniel-KM committed:
Called js function for advanced search only when needed.

Daniel-KM committed:
Genericized main search results.

Daniel-KM committed:
Normalized size of the popover advanced form.

Daniel-KM committed:
Fixed the queue of js.

Daniel-KM committed:
Integrated the breadcrumb trail.

Daniel-KM committed:
Upgraded for the last release of contribution.

Daniel-KM committed:
Added featured and recent items and featured collection.

Daniel-KM committed:
Added a warn about the use of a Geolocation fork.

Daniel-KM committed:
Integrated public open taxonomy.

Daniel-KM committed:
Added an option to display Geolocation map on home page.

Daniel-KM committed:
Added the sort links to collections/browse and items/browse.

Daniel-KM committed:
Themed files/show.

Daniel-KM committed:
Simplified the search form.

Daniel-KM committed:
Updated footer bootstrap

Daniel-KM committed:
Simplified caption.

Daniel-KM committed:
Fixed citation in items/browse.

Daniel-KM committed:
Integrated simple pages.

Daniel-KM committed:
Normalized bootstrap columns.

Daniel-KM committed:
Reset the search form for Omeka 2.5.

Daniel-KM committed:
Integrated pages for Contribution.

Daniel-KM committed:
Fixed inline view of Bootstrap for Zend.

Daniel-KM committed:
Included a class to redecorate built forms.

Daniel-KM committed:
Added a view helper path.

Daniel-KM committed:
Reorganized config.ini.

Daniel-KM committed:
Removed body background.

Daniel-KM committed:
Added carousel on items/browse.

Daniel-KM committed:
Used bootstrap styles for flash.

Daniel-KM committed:
Improved search form for specific record types.

Daniel-KM committed:
Displayed home page text instead of the title.

Daniel-KM committed:
Integrated Exhibits pages.

Daniel-KM committed:
Normalized queue of css.

Daniel-KM committed:
Integrated login pages.

Daniel-KM committed:
Cleaned footer.

Daniel-KM committed:
Used a specific partial for menu.

Daniel-KM committed:
Fixed advanced search.

Daniel-KM committed:
Standardized title of pages with a secondary nav.

Daniel-KM committed:
Overwritten helpers for FormLabel, FormText, FormSelect, ElementForm and ElementInput.

Daniel-KM committed:
Reindented style.css.

Daniel-KM committed:
Fixed contribution form for Geolocation.

Daniel-KM committed:
Finalized advanced search form.

Daniel-KM committed:
Added link to item in files/show.

Daniel-KM committed:
Added glyphicon for the title of the pages User and Contribution.

Daniel-KM committed:
Overwritten helper for FormButton.

Daniel-KM committed:
Updated to remove warnings during upgrade to Omeka S.

Daniel-KM committed:
Set minimum version to Omeka 2.2.2.

Daniel-KM committed:
Fixed hook "public_collections_show".

Daniel-KM committed:
Fixed custom functions to allow php 5.

Daniel-KM committed:
Fixed items/single to allow Omeka 2.4.

Commit:
Updates to Omeka Bootstrap Theme

`collections/browse.php`:
- Updated sort-links to more closely match Bootstrap styling
- When no associated image for a collection, now displays the
`no-file.png` replacement image
- Updated alignment of description and link to collection so it
left-aligns with collection title instead of wraps below or aligns
right, respectively

`collections/show.php`:
- Updated old bootstrap class of `img-polaroid` with bootstrap3 class of
`img-thumbnail`

`common/breadcrumb.php`:
- Updated breadcrumbs to provide a more correct breadcrumb trail
depending on content and settings
- Added Exhibit Builder `show-item` breadcrumb capability -- full
breadcrumb to first matching Exhibit related to currently viewed item
(if item belongs to multiple exhibits it just uses the first returned
Exhibit from the database since we can't know, via PHP, which Exhibit
the user came from)

`common/header.php`:
- Removed extra orphaned `<div>`

`common/show-selected-metadata.php`:
- Added this file for the `items/show.php` rendering, file seemed to be
missing from repo

`css/style.css`:
- Added helpful classes for compatibility with OpenSeaDragon and Zoomit
Omeka plugins
- Added custom theme styling for `#sort-links` due to change in
`collections/browse.php` to use Bootstrap styling of sort links
- Changed `#search-filters li` styling to better match other theme
content styles

`custom.php`:
- Added optional parameter (and functionality with the parameter) to
`bootstrap_breadcrumb_exhibit_page()` function to allow for the last
element in the breadcrumb trail to also be hyperlinked (used for Exhibit
Item Show page)
- Added `custom_files_for_item()` and `custom_file_markup()` to allow
images to be displayed with HTML5 `<figure>` element markup and
associated filename
- Added `related_exhibit_page()` function as helper to item-show
breadcrumb creation

`items/browse.php`:
- Updated sort-links to more closely match Bootstrap styling
- Updated old bootstrap class of `img-polaroid` with bootstrap3 class of
`img-thumbnail`

`items/show.php`:
- Added `form-group` class to DIV as a hackish fix for better style
matching in two places
- Changed "Files" header to "File" OR "Files" depending on number of
matching items; implemented custom_files_for_item function from updated
theme's `custom.php`

`items/single.php`:
- When no associated image for a collection, now displays the
`no-file.png` replacement image

Commit:
Fixing two new caught issues

`collections/show.php`:
- Closing DIV placed outside of if/else block could cause HTML
validation error (orphaned `</div>`)

`css/style.css`:
- Fixes potential layout issue with placement/arrangement of
popover-form labels for advanced search (labels and related checkboxes
now appear correctly aligned (instead of staggered due to float)

`search/search-form.php`:
- Fixed one spot causing HTML validation errors from duplicate IDs
(there are likely more)

Daniel-KM committed:
Added Brendon Kozlowski in the authors.

Daniel-KM committed:
Fixed breadcrumb in simple pages.

Daniel-KM committed:
Replaced "'item'" by "$item".

Daniel-KM committed:
Fixed global search with simple page.

Daniel-KM committed:
Removed a old note.

Daniel-KM committed:
Fixed docblocks.

Daniel-KM committed:
Removed a useless javascript in header.

Daniel-KM committed:
Fixed missing </div> in reference.

Daniel-KM committed:
Removed duplicate discovery links.

Daniel-KM committed:
Fixed carousel with tag in items/browse.

Daniel-KM committed:
Updated links.

Daniel-KM committed:
Fixed no script warn.

Daniel-KM committed:
Managed plugin LocaleSwitcher.

Daniel-KM committed:
Fixed google analytics.
